<?xml version="1.0"?>
<launch>
  <node name="inner_loop_node" type="inner_loop_node" pkg="quad_control">
    <remap from="/odometry"         to="/hummingbird/odometryNED/eta"/>
    <remap from="/wrenchEstimator"  to="/hummingbird/wrenchEstimator/torque"/>
    <remap from="/eta_d"            to="/hummingbird/trajectory"/>
    <remap from="/mu_hat"           to="/hummingbird/mu_hat"/>
    <remap from="/wrenchNED"        to="/hummingbird/command/wrenchNED"/>
    <remap from="/error_pose"       to="/hummingbird/error/pose/angular"/>
    <remap from="/error_twist"      to="/hummingbird/error/twist/angular"/>
    <remap from="/error_wrench"     to="/hummingbird/error/wrench/angular"/>
    <param name="mass"  type="double"  value="0.68"/>
    <param name="Ixx"   type="double"  value="0.007"/>
    <param name="Iyy"   type="double"  value="0.007"/>
    <param name="Izz"   type="double"  value="0.012"/>
    <param name="Kp"    type="double"  value="100"/>
    <param name="Kd"    type="double"  value="10"/>
    <param name="Kd"    type="double"  value="10"/>
    <param name="Ts"    type="double"  value="0.001"/>
  </node>
</launch>